testapi.pm

```diff
@@ -1383,7 +1383,7 @@ sub select_console {
         if ($autotest::last_milestone) {
             push(@{$autotest::last_milestone->{activated_consoles}}, $testapi_console);
         }
-        $testapi::distri->activate_console($testapi_console);
+        $testapi::distri->activate_console($testapi_console, @args);
```

Comment:
wait, you forward the same args into activate_console as well as console_selected? I wonder if this is not confusing.

Reply:
@okurz That's the thing I also do not like, but it will be more mess if we try to get two hashes of settings and separate them. And in our distri specific implementations we still can have parameters which are valid for both callback functions.
@coolo I actually checked some other PRs and they do not contain changes to version. But it makes sense.